assets/src/edit-story/elements/karma/multipleElementsHistory.js

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { Fixture } from '../../karma';
+import { useInsertElement } from '../../components/canvas';
+import createSolid from '../../utils/createSolid';
+
+const generateHelpers = (fixture) => {
+  const fullBleed = fixture.container.querySelector(
+    '[data-testid="fullbleed"]'
+  );
+  const fullBleedBB = fullBleed.getBoundingClientRect();
+  const safezone = fixture.container.querySelector('[data-testid="safezone"');
+  const safezoneBB = safezone.getBoundingClientRect();
+
+  const generateHelpersForBB = (bb) => ({
+    dnd: async (fromX, fromY, toX, toY) => {
+      await fixture.events.mouse.seq([
+        {
+          type: 'move',
+          x: bb.x + fromX,
+          y: bb.y + fromY,
+        },
+        {
+          type: 'down',
+        },
+        {
+          type: 'move',
+          x: bb.x + toX,
+          y: bb.y + toY,
+          options: { steps: 12 },
+        },
+        {
+          type: 'up',
+        },
+      ]);
+    },
+  });
+
+  return {
+    fullBleed: generateHelpersForBB(fullBleedBB),
+    safeZone: generateHelpersForBB(safezoneBB),
+  };
+};
+
+// eslint-disable-next-line no-unused-vars
+function sleep(ms) {
+  return new Promise((resolve) => setTimeout(resolve, ms));
+}
+jasmine.DEFAULT_TIMEOUT_INTERVAL = 60 * 1000;
```

Comment:
This will be removed, right?

Reply:
Yes, it will be removed when I push fixed for small canvas.

Reply:
K. The new tools for debugging are also now available. See https://github.com/google/web-stories-wp/blob/master/karma/README.md#debugging-karma-tests